packages/nextra-theme-docs/css/styles.css

```diff
@@ -23,22 +23,14 @@ body {
   @apply nx-w-full nx-bg-white dark:nx-bg-dark dark:nx-text-gray-100;
 }
 
-a,
-summary,
-button,
-input,
-[tabindex]:not([tabindex='-1']) {
```

Comment:
I couldn’t find any `input` elements that needed this, or `[tabindex]` elements in the codebase. I’m guessing these were just anticipatory styles?

All the `<a>`, `<summary>`, and `<button>` tags generated by Nextra got the `.next-focusable` class.